### DISCUSSION TITLE
tpw-widget defaults to accordion on Safari

### DISCUSSION THREAD

**Discussion by mattwren88** [tabpanelwidget/tabpanelwidget#18] [Q&A] [answered]
I placed the CDN links of TabPanel onto a page to test it out and it works great but when I view it on Safari on MacOS it goes directly to tpw-accordion. I have the polyfill placed so I'm not sure why it's different on this browser. Your website displays it as tabs first as well.

**Comment by thierryk** [comment, marked as answer]
I just checked the website in Safari 15.6.1/MacOS Monterey and everything seems to be working as expected.

> Your website displays it as tabs first as well

What do you mean by that? This is how it's supposed to work. Or do you mean it shows as TabPanel but then switch to Accordion on its own? I'm sorry but I'm not sure I understand the problem here.
Do you have a page I can look at?

**Reply by mattwren88** [reply]
Sorry I think I worded that confusingly. I'm just saying in my instance it is defaulting to accordion on Safari and not as tabs first. Your site works as intended but mine isn't
https://www.scranton.edu/about/pointsofpride/index.shtml

**Reply by thierryk** [reply]
Thanks for providing a link to your widget.

If it does not show as a TabPanel it is because **things are working as expected**.

It is the `max-width` in this rule:

```
.row {
    width: 100%;
    max-width: 1280px;
    margin: 0 auto;
}
```

that prevents the widget from showing as a TabPanel. That's because there is not enough room to display the "tabs" side-by-side.
If you increase the `max-width` value, that "accordion" widget should become a TabPanel.

**Reply by mattwren88** [reply]
Ahh I gotcha. that makes sense and it's good that it still works well. Was just confused why it was only happening on Safari.

**Reply by thierryk** [reply]
It displays the same in Chrome. 
On Firefox it shows as a TabPanel.

If you want to show a TabPanel across the board you may want to edit the value in this declaration:

```
font-size: calc(1em * 1.1);
```

A simple `1em` font-size should do the trick.

**Reply by mattwren88** [reply]
Yes this is it, thank you!

**Reply by thierryk** [reply]
You're very welcome!
